packages/api/src/folder/controller.test.ts

```diff
@@ -0,0 +1,134 @@
+import type { Request, NextFunction } from "express";
+import { logger } from "@stela/logger";
+import request from "supertest";
+import { app } from "../app";
+import { db } from "../database";
+import {
+  extractUserEmailFromAuthToken,
+  verifyUserAuthentication,
+} from "../middleware";
+
+jest.mock("../database");
+jest.mock("../middleware");
+jest.mock("@stela/logger");
+
+const setupDatabase = async (): Promise<void> => {
+  await db.sql("fixtures.create_test_folders");
+};
+
+const clearDatabase = async (): Promise<void> => {
+  await db.query(
+    `TRUNCATE 
+       folder CASCADE`
+  );
+};
+
+describe("patch folder", () => {
```

Comment:
For complete coverage, we should have a test case for if the SELECT after the UPDATE fails, and another for if it returns zero rows.